src/main/java/com/criticalsoftware/UserRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.criticalsoftware;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+
+//User Data Request
+public class UserRequest {
+    @NotBlank(message = "Name may not be blank")
+    private String name;
```

Comment:
Not null missing.

Reply:
The @notblank ensures that a String is not null, is not empty, and has at least one visible character.